src/devices/thermostat.cpp

```diff
@@ -2894,6 +2935,14 @@ bool Thermostat::set_temperature(const float temperature, const uint8_t mode, co
             offset          = 0;
             factor          = 1;
             break;
+        case HeatingCircuit::Mode::NOREDUCE:
+            offset = 12;
+            factor = 1;
+            break;
+        case HeatingCircuit::Mode::REDUCE:
+            offset = 9;
+            factor = 1;
+            break;
```

Comment:
For this two cases you should add a line `validate_typeid = set_typeid;` to readback the set-telegram for updating the change.

Reply:
@MichaelDvP I have done the required changes together with wwprio for RC310

Reply:
set_typeid defaults to set_typeids(), redundand to set it again.